xla/service/elemental_ir_emitter.cc

```diff
@@ -2921,10 +2921,12 @@ absl::StatusOr<llvm::Value*> ElementalIrEmitter::EmitElementalDot(
         "Algorithm not supported by the ElementalIrEmitter: %s",
         PrecisionConfig::Algorithm_Name(hlo->precision_config().algorithm())));
   }
+#ifdef GOOGLE_CUDA
   const HloDotInstruction* dot = Cast<HloDotInstruction>(hlo);
   if (dot->sparse_operands()) {
     return Unimplemented("Sparse dot is supported by Triton emitter only.");
   }
+#endif
```

Comment:
IIUC, sparse dot will require `[AddGemmFusionAutotuningPasses](https://github.com/openxla/xla/blob/main/xla/service/gpu/gemm_fusion_autotuner.cc)` and that not only requires triton auotune, but also cudnn fusion front as well.